web/src/app/components/bar-chart/bar-chart.component.ts

```diff
@@ -208,10 +212,9 @@ export class BarChartComponent implements AfterViewInit {
     const element = this.barChartContainer.nativeElement;
     const elementWidth = element.clientWidth;
     const elementHeight = element.clientHeight;
-    const marginFocus = {top: 60, right: 90, bottom: 150, left: 70};
-    const marginBrush = {top: 370, right: 90, bottom: 20, left: 70};
+    const marginFocus = {top: 60, right: 90, bottom: 150, left: 110};
+    const marginBrush = {top: 380, right: 90, bottom: 20, left: 110};
```

Comment:
```suggestion
    const marginFocus = { top: 60, right: 90, bottom: 150, left: 110 };
    const marginBrush = { top: 380, right: 90, bottom: 20, left: 110 };
```

Reply:
I assume we want to follow clang-formatter here.

Reply:
Yup.